pywhat/Data/regex.json

```diff
@@ -426,6 +427,28 @@
           "Identifiers",
           "Credentials"
        ]
+    },
+      {
+      "Name": "Recent Unix Timestamp",
+      "Regex": "^[0-9]{10}$",
+      "plural_name": false,
+      "Description": "Seconds elapsed since unix epoch: 1970, between year 2001 and 2286",

```

Comment:
@SkeletalDemise  @piatrashkakanstantinass @amadejpapez  how do we feel about these descriptions? We generally try to avoid them but I can understand Unix Timestamp is confusing.

Just wanted to get ya thoughts :)

Reply:
I think it should stay because as you said Unix Timestap is confusing, so this description might come in useful :)

Reply:
I think we should actually write the exact time in description. e.g. phone codes